src/pages/Home/Home.page.tsx

```diff
@@ -44,6 +46,25 @@ const Sponsors = sponsors.map((sponsor, i) => {
 });
 
 const HomePage = () => {
+    const [rsvpStatus, setRsvpStatus] = useState("Not RSVP'd");
+    const functions = getFunctions();
+
+    useEffect(() => {
+        const fetchRSVPStatus = async () => {
+            const fetchRSVPStatus = httpsCallable(functions, "getRVSPStatus");
+            try {
+                const result = await fetchRSVPStatus();
+                console.log("RSVP status:", result.data);
+                setRsvpStatus(result.data ? "RSVP'd" : "Not RSVP'd");
+            } catch (error) {
+                console.error("Error fetching RSVP status:", error);
+                setRsvpStatus("Error fetching status");
+            }
+        };
+
```

Comment:
we can use `currentUser.rsvpVerified` from the auth provider.

Reply:
You can check out `routes.provider.tsx` how i use it

Reply:
updated in newest commit